browser/main/NoteList/index.js

```diff
@@ -124,27 +144,49 @@ class NoteList extends React.Component {
     }
   }
 
+  focusNote (selectedNoteKeys, noteKey) {
+    let { router } = this.context
+    let { location } = this.props
```

Comment:
plz use `const` instead of `let` 🙏

Reply:
(we know there are many `let` in existing code. but it doesn't make sense, so we want to use `const` as possible as we can)